### DISCUSSION TITLE
question about default settings in firmware 427-BLT and 422-BLT

### DISCUSSION THREAD

**Discussion by nwpayton** [mriscoc/Ender3V2S1#451] [Q&A]
I am wondering if Z_SAFE_HOMING is enabled by default in these precompiled firmware's. I was gonna compile my own (i had VSS open and was looking through the config files) but I wasn't sure what settings needed changed to be as close to default 422-BLT or 427-BLT as it could be. so if I can avoid compiling my own then that would be good and I would only need to do that if safe homing is off.

**Comment by mriscoc** [comment]
You can get the configuration files from: https://github.com/mriscoc/Special_Configurations
